contributing/development/compiling/compiling_for_windows.rst

```diff
@@ -214,6 +217,17 @@ Optionally, you can compile with the following for additional features:
 .. note:: If you use a preview version of the Agility SDK, remember to enable
           developer mode in Windows; otherwise it won't be used.
 
+.. note:: If you want to use a PIX with MinGW build, navigate to PIX runtime
+          directory and use the following commands to generate import library::
+
+            # For x86-64:
+            gendef ./bin/x64/WinPixEventRuntime.dll
+            dlltool --machine i386:x86-64 --no-leading-underscore -d WinPixEventRuntime.def -D WinPixEventRuntime.dll -l ./bin/x64/libWinPixEventRuntime.a
+
+            # For ARM64:
+            gendef ./bin/ARM64/WinPixEventRuntime.dll
+            dlltool --machine arm64 --no-leading-underscore -d WinPixEventRuntime.def -D WinPixEventRuntime.dll -l ./bin/ARM64/libWinPixEventRuntime.a
```

Comment:
This should be indented by spaces not tabs I think

Reply:
There are no tabs in the source.

Reply:
Strange the git edit showed them up at first, my bad 🙂